### ISSUE TITLE
Schedule with integer parameter

### ISSUE THREAD

**Issue by jcassette** [espressif/esp-rainmaker-android#19]
Hello

First, thank you for developing this amazing project!

I have created a device with the "Outlet" type. It contains a power parameter and an integer parameter.

From the "Devices" tab of the Android app I can modify both parameters:
```
I (7709752) esp_rmaker_param: Received params: {"Outlet 1 (IO5)":{"Power":true}}
I (7709752) esp_rmaker_param: Reporting params: {"Outlet 1 (IO5)":{"Power":true}}
I (7712422) esp_rmaker_param: Received params: {"Outlet 1 (IO5)":{"Power":false}}
I (7712422) esp_rmaker_param: Reporting params: {"Outlet 1 (IO5)":{"Power":false}}
I (7718052) esp_rmaker_param: Received params: {"Outlet 1 (IO5)":{"Timer":10}}
I (7718052) esp_rmaker_param: Reporting params: {"Outlet 1 (IO5)":{"Timer":10}}
I (7728052) esp_rmaker_param: Reporting params: {"Outlet 1 (IO5)":{"Power":false}}
I (7728052) esp_rmaker_param: Reporting params: {"Outlet 1 (IO5)":{"Timer":0}}
```

However when I create a schedule the integer parameter is always sent equal to 0.
For example here I try to create a schedule with Power = true and Timer = 10:
```
I (7741912) esp_rmaker_param: Received params: {"Schedule":{"Schedules":[{"operation":"edit","id":"7SGW","name":"Aaa","triggers":[{"d":0,"m":969}],"action":{"Outlet 1 (IO5)":{"Power":true,"Timer":0}}}]}}
```

Can you help please?

Julien

**Comment by khushbushah2302** [comment]
Hi @jcassette ,
Thanks for reporting this issue.
Will check and fix it.

**Comment by khushbushah2302** [comment]
Hi @jcassette ,

Fixed the issue in latest app (version 2.6.1 - 5ecc246).
Commit id : 5ecc246ac1cecc35e2f7ddfae446117f1377d85c
